src/utils/utils.ts

```diff
@@ -171,7 +171,7 @@ export const utf8ToBytes32 = (str: string): string => {
 }
 
 // turns multi-line query into single line and removes extra spaces
-export const cleanRequest = (query: any) => query.replace(/\n+ /g, '').replace(/\s+ /g, ' ')
+export const cleanRequest = (query: string): string => query.replace(/\n+ /g, '').replace(/\s+ /g, ' ')
```

Comment:
RegEx is very sensitive and the extra spaces in `/\n+ /g` are counted as well. Use this approach instead to remove all new lines, temporarily add a space, and then purge all spaces with more than two occurrences in a row -> `export const cleanRequest = (query: string): string => query.replace(/\n/g, ' ').replace(/\s{2,}/g, ' ')`